src/app/parcels/form/ParcelForm.tsx

```diff
@@ -187,7 +187,15 @@ const ParcelForm: React.FC<ParcelFormProps> = ({
         setSubmitDisabled(true);
         let inputError;
         if (fields.shippingMethod === "Collection") {
-            inputError = checkErrorOnSubmit(formErrors, setFormErrors);
+            inputError = checkErrorOnSubmit(formErrors, setFormErrors, [
```

Comment:
Am a little confused as to why this will make a difference? 
Lines 215 and 216 of formFunctions.ts
` for (const [errorKey, error] of Object.entries(errorType)) {
        if (!keysToCheck || keysToCheck.includes(errorKey)) {`
if keysToCheck is empty, then all the keys (in errorType) will be checked. Doesn't errorType contain the same set of 7 keys that are now in the added array. So in both cases all 7 keys will be checked?

Reply:
yeah good point. the error was occurring because another key was accidentally added to errorType. I've fixed this, and removed the above change.